corehq/apps/app_manager/models.py

```diff
@@ -6247,6 +6247,23 @@ def to_json(self, app_names):
         return LatestEnabledBuildProfileSerializer(self, context={'app_names': app_names}).data
 
 
+class ApplicationReleaseLog(models.Model):
+    domain = models.CharField(max_length=255, null=False, default='')
+    timestamp = models.DateTimeField(auto_now_add=True)
+    is_released = models.BooleanField()
```

Comment:
can we make this `action` instead? I am thinking for now we just have "Released" and "In-Test" but if we want to track any other action in future, we would need to modify the model, like say for "delete"? 
This makes me think if we should track delete as well?

Reply:
Yeah, this makes sense, better to have an `action` field than `is_released`, that will make it flexible to track other actions on the app releases.